internal/flags/flags.go

```diff
@@ -15,32 +15,59 @@
+	Provider                        = "provider"                        // Provider flag to set the cloud provider
+	Region                          = "region"                          // Region flag
+	RegionShort                     = "r"                               // RegionShort flag
+	Members                         = "members"                         // Members flag
+	MembersShort                    = "m"                               // MembersShort flag
+	InstanceSize                    = "instanceSize"                    // InstanceSize flag
+	DiskSizeGB                      = "diskSizeGB"                      // DiskSizeGB flag
+	MDBVersion                      = "mdbVersion"                      // MDBVersion flag
+	Backup                          = "backup"                          // Backup flag
+	Username                        = "username"                        // Username flag
+	Password                        = "password"                        // Password flag
+	Email                           = "email"                           // Email flag
+	FirstName                       = "firstName"                       // FirstName flag
+	LastName                        = "lastName"                        // LastName flag
+	Role                            = "role"                            // Role flag
+	Type                            = "type"                            // Type flag
+	Comment                         = "comment"                         // Comment flag
+	Page                            = "page"                            // Page flag
+	Limit                           = "limit"                           // Limit flag
+	File                            = "file"                            // File flag
+	FileShort                       = "f"                               // File flag
+	Force                           = "force"                           // Force flag
+	WhitelistIP                     = "whitelistIp"                     // WhitelistIP flag
+	Event                           = "event"                           // EventTypeName flag
+	Enabled                         = "enabled"                         // Enabled flag
+	MatcherFieldName                = "matcherFieldName"                // MatcherFieldName flag
+	MatcherOperator                 = "matcherOperator"                 // MatcherOperator flag
+	MatcherValue                    = "matcherValue"                    // MatcherValue flag
+	MetricThresholdMetricName       = "metricThresholdMetricName"       // MetricThresholdMetricName flag
+	MetricThresholdOperator         = "metricThresholdOperator"         // MetricThresholdOperator flag
+	MetricThresholdThreshold        = "metricThresholdThreshold"        // MetricThresholdThreshold flag
+	MetricThresholdUnits            = "metricThresholdUnits"            // MetricThresholdUnits flag
+	MetricThresholdMode             = "metricThresholdMode"             // MetricThresholdMode flag
```

Comment:
- `--metricThresholdMetricName`, double metric, what about just `--metricName` or just `--metric`
- `--metricThresholdOperator`, maybe just `metricOperator`
- `--metricThresholdThreshold `, double threshold, what about `metricThreshold` ?
- `--metricThresholdUnits`, `--metricThresholdMode` to keep things in line maybe also just `metricUnits` and `--metricMode`

Reply:
not needed but consider changing the const name to match